lib/messages/abstract-issue.js

```diff
@@ -47,7 +47,7 @@ class AbstractIssue extends Message {
     } else {
       actor = this.abstractIssue.user.login;
     }
-    return `${subject} ${predicate} by ${actor}`;
+    return `${subject} ${predicate.replace(/_/g, ' ')} by ${actor}`; // e.g. replace 'ready for review' with 'ready_for_review'
```

Comment:
```suggestion
    return `${subject} ${predicate.replace(/_/g, ' ')} by ${actor}`; // e.g. replace 'ready_for_review' with 'ready for review'
```

Reply:
Good catch!

I think the Codecov failed because Travis said the CI failed initially and I couldn't get Codecov to re-run it's checks. We'll see what happens on this commit! ❤️